mlir/lib/Dialect/QCO/Transforms/Optimizations/QuaternionMergeRotationGates.cpp

```
@@ -0,0 +1,629 @@
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * Licensed under the MIT License
+ */
+
+#include "mlir/Dialect/QCO/IR/QCOInterfaces.h"
+#include "mlir/Dialect/QCO/IR/QCOOps.h"
+#include "mlir/Dialect/QCO/Transforms/Passes.h"
+
+#include <llvm/ADT/TypeSwitch.h>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)